### DISCUSSION TITLE
Is it possible to print or render some styled text without the reset tags being automatically added?

### DISCUSSION THREAD

**Discussion by ryran** [Textualize/rich#2747] [Q&A]
For a certain part of my app, I have a subprocess that needs to stream its output directly to the terminal (vs capturing it and re-printing it via rich console), so before I launch the subprocess, I want to send some color escapes to the terminal.

Without rich, it's easy enough, but I'm already allowing users to customize output colors via env vars, so I can't just do a simple `print('\033[2;34m')` (for example), since I need to parse the user-provided style for this category of output.

So to summarize: I'm looking for a way in rich to parse a style (with or without some text) and output the ANSI escapes without the closing/reset. Possible?
